### DISCUSSION TITLE
Disable magit auto revert mode in tramp by default

### DISCUSSION THREAD

**Discussion by ParetoOptimalDev** [magit/magit#4817] [Ideas]
I think this might make a more sensible default based on my experiences in the past.

I was reminded by this recent reddit thread: https://www.reddit.com/r/emacs/comments/yw3gpx/magit_autorevert_and_tramp/

Note: searched for similar discussions with link by checkbox and the more general query:

https://github.com/magit/magit/discussions?discussions_q=revert

**Comment by tarsius** [comment]
This is the default now.

(Those who want to `auto-revert-remote-files`, have to set that variable.)
